numpy/core/_add_newdocs.py

```diff
@@ -8009,22 +8022,65 @@ def luf(lamdaexpr, *args, **kwargs):
 
 add_newdoc('numpy.core.numerictypes', 'float128',
     """
-    128-bit floating-point number. Character code: 'g'. C long float
+    128-bit floating-point number. Character code: ``'g'``. C long float
     compatible.
```

Comment:
c long float (or long double?). Wonder if it should warn here that it is not an IEEE quad float type. And the same text is  true for float96, also... But maybe this is not the place to get into those details. Not sure how prominent this shows up also on the online docs.